packages/hint/src/lib/hint-context.ts

```diff
@@ -123,7 +123,7 @@ export class HintContext<E extends Events = Events> {
     }
 
     /** Reports a problem with the resource. */
-    public report(resource: string, message: string, options: ReportOptions = {}) {
+    public report(resource: string, message: string, options: ReportOptions = { severity: Severity.default }) {
         const { codeSnippet, element, severity = Severity.warning } = options;
```

Comment:
I think you don't need this `severity = Severity.warning`, because severity is going to be set at least to `.default`

Reply:
btw, do you want the default value to be `warning` or `default`?

Reply:
I think it should be `Default` but maybe I should just remove the default value directly

Reply:
If severity is going to be mandatory, I don't think you need any default value.

Reply:
I've changed that. Can you take another look?